src/themes/themes.js

```diff
@@ -6,12 +6,19 @@ define(['jquery', 'windows/windows', 'highstock', "jquery-growl"], function($, w
     var win = null;
 
     /*Set theme from local storage*/
-    var themeName = local_storage.get("theme");
+    var themeName = local_storage.get("theme"),
+        custom_theme = local_storage.get("custom_theme");
     themeName = themeName && themeName.name;
     if (themeName) {
         require(['lib/highstock/themes/' + themeName]);
+    } else if(custom_theme){
+        console.log(custom_theme);
+        Highcharts.setOptions(custom_theme);
     }
 
+    //For custom theme.
+    require(['themes/custom_theme/custom_theme']);
```

Comment:
@4p00rv This is incorrect. You require where you need. You just should not require something if you are not going to use it.